src/blocks/scratch3_procedures.js

```diff
@@ -13,46 +13,39 @@ class Scratch3ProcedureBlocks {
      */
     getPrimitives () {
         return {
-            procedures_definition: this.definition,
```

Comment:
Maybe add a comment in place of this line for why procedures_definition isn't mentioned as a one of the 'primitives'

Reply:
On second thought, what if we made `procedures_definition: null` an entry here instead of removing it? In general, I don't like having `undefined` be something meaningful...

Reply:
Ah. Hmm. Setting this to null, we need to change a test in the runtime method that register's primitives. It tries to bind null like its a function. That is to say the register primitives algorithm only accepts functions currently.

I'd guess we'll need to change other things in there.

Reply:
I'm talking about https://github.com/LLK/scratch-vm/pull/2145/files#diff-38d875e1d50b3158dc96404b65275a54R715.